### PR TITLE
golangci-lint round #3

### PR DESCRIPTION
**Pull request by baude**
this is the third round of preparing to use the golangci-lint on our
code base.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/trust_set_show.go [hunk 1]
```diff
@@ -254,15 +254,16 @@ func getPolicyJSON(policyContentStruct trust.PolicyContent, systemRegistriesDirP
 			policyJSON[repo]["type"] = repoval[0].Type
 			policyJSON[repo]["transport"] = transname
 			keyarr := []string{}
-			uids := []string{}
+			// TODO - uids is never used.  i have no idea about the intent; commenting out to fix later
```
Comment (rhatdan) [inline cmd/podman/trust_set_show.go]: @QiWang19 PTAL at these.  I believe they can just be removed.

File: pkg/apparmor/apparmor_linux.go [hunk 2]
```diff
@@ -87,20 +88,24 @@ func InstallDefault(name string) error {
 
 	cmd := exec.Command("apparmor_parser", "-Kr")
 	pipe, err := cmd.StdinPipe()
+	defer func() {
```
Comment (rhatdan) [inline pkg/apparmor/apparmor_linux.go]: This defer is in the wrong place, it should happen after the `if err` check.

File: cmd/podman/trust_set_show.go (hunk 1 above)
Comment (QiWang19) [reply cmd/podman/trust_set_show.go]: Yes. This can be removed.

File: cmd/podman/shared/create.go [hunk 3]
```diff
@@ -804,7 +804,7 @@ func makeHealthCheckFromCli(c *GenericCLIResults) (*manifest.Schema2HealthConfig
 	if err != nil {
 		return nil, errors.Wrapf(err, "invalid healthcheck-timeout %s", inTimeout)
 	}
-	if timeoutDuration < time.Duration(time.Second*1) {
+	if timeoutDuration < time.Duration(1) {
```
Comment (jwhonce) [inline cmd/podman/shared/create.go]: Does the linter accept (time.Duration(1) * time.Second)? I hate to lose the units here.

File: .idea/libpod.iml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mheon) [inline .idea/libpod.iml]: All this stuff in .idea should be gitignore'd

**Pull request merged**